spec/requests/api/v1/npq_participants_spec.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "NPQ Participants API", type: :request, with_feature_flags: { participant_data_api: "active" } do
+  describe "GET /api/v1/npq-participants" do
+    let!(:default_schedule) { create(:schedule, name: "ECF September standard 2021") }
+    let(:npq_lead_provider) { create(:npq_lead_provider) }
+    let(:cpd_lead_provider) { create(:cpd_lead_provider, npq_lead_provider: npq_lead_provider) }
+    let(:token) { LeadProviderApiToken.create_with_random_token!(cpd_lead_provider: cpd_lead_provider) }
+    let(:bearer_token) { "Bearer #{token}" }
+
+    context "when authorized" do
+      before do
+        default_headers[:Authorization] = bearer_token
+      end
+
+      before :each do
+        list = []
+        list << create_list(:npq_validation_data, 3, npq_lead_provider: npq_lead_provider, school_urn: "123456")
```

Comment:
can this not be `list =` and then we can remove the `flatten` below?

Reply:
done